hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/hedera/utils/AddressBookUtils.java

```diff
@@ -70,6 +74,28 @@ public static Map<Long, Bytes> certsFor(@NonNull final String configTxt) {
                 }));
     }
 
+    /**
+     * Returns a deterministic, unique gossip CA certificate for the given node id.
+     *
+     * @param nodeId the node id
+     * @return the certificate bytes
+     */
+    public static byte[] gossipCaCertificateForNodeId(final long nodeId) {
+        return GOSSIP_CERT_CACHE.computeIfAbsent(nodeId, id -> {
+            final var nodeKey = NodeId.of(id);
+            try {
+                final var keysAndCerts =
+                        CryptoStatic.generateKeysAndCerts(Set.of(nodeKey), null).get(nodeKey);
```

Comment:
Do we have to invoke the entire keys/certs generation process to produce a working cert? The debugger gets trapped inside `generateKeysAndCerts(...)` every time